### DISCUSSION TITLE
Adding new languages

### DISCUSSION THREAD

**Discussion by ferferga** [jellyfin/jellyfin-vue#2184] [UI & UX]
Hello!

Thank you for your interest in translating Jellyfin Vue! I have a few *"considerations"* for adding new languages:

- **That you have some interest in keeping that translation alive:** You're not "bounded" by any means and are free to help as long as you desire to but, in an **ideal world**, this is what we'd like:
    - That you translate a fairly good amount of strings of that new language (not just 3 strings)
    - No empty languages: they are misleading to users (as they will appear listed in the UI), thinking that the app is translated to their languages when it's not,
    - That you come by from time to time to check if there's something new.

- **For languages with cultures (for example, Spanish with Spain/LATAM, Brazilian Portuguese, US/UK English, etc):**
    I'd like to know if there is a strong reason to keep separate cultures for the language you want to add. For example, in the case of Serbian, **it makes total sense to have support for the latin alphabet and cyrillic!**
    However, in Spanish, most strings are the same in both variants and the ones that are different can be written in **a neutral manner that sounds good for all dialects** (that's what international alphabetization is for).

    Extra languages add a maintenance burden and increase the bundle size of the client **for all users, regardless of the language they will use**, so all reductions we can make in this area are not to be taken lightly.

Post a comment here (or send it through weblate' add new language form) with the language you want to be added to the project and a brief explanation of why it needs different cultures (if applicable). Anything else you see worth mentioning is welcome too!

**Comment by yarons** [comment]
Maybe we can set some inclusion criteria that will also cover long term commitment (over 80 percent), this way if some translation is over 80 percent it will be included, if more original strings are added without maintaining the translation it might drop below 80 percent thus excluded, it will be available on Weblate regardless.
